script/make-reference-release.bash

```diff
@@ -29,22 +29,61 @@
 
 set -euxo pipefail
 
-mkdir -p build
+readonly OT_PLATFORMS=(nrf52840 efr32mg1 efr32mg12 efr32mg13 efr32mg21)
 
-echo "REFERENCE_RELEASE_TYPE=${REFERENCE_RELEASE_TYPE?}"
+main()
+{
+    if [[ $# == 0 ]]; then
```

Comment:
Since this script previously did not require a platform for backward compatibility we should either default to nrf52840 or build both nrf52840 and efr32mg12 when no platform is specified.  I think I'd prefer building both.

Reply:
Good point. I changed it to build both